src/v3/src/components/IdentifierContainer/IdentifierContainer.tsx

```diff
@@ -35,45 +34,27 @@ const IdentifierContainer: UISchemaElementComponent<{
       maxWidth={1}
       className={mainContainerClasses}
     >
-      <Box
-        flex="auto"
-        flexDirection="row"
-        flexGrow="0"
-        paddingX={4}
-        paddingY={2}
-        sx={{
-          borderRadius: '36px',
-          backgroundColor: tokens.HueNeutral50,
-          textOverflow: 'ellipsis',
-          whiteSpace: 'nowrap',
-          overflow: 'hidden',
-        }}
-      >
-        <Box
-          component="span"
-          sx={{
-            color: tokens.PalettePrimaryMain,
-            verticalAlign: 'middle',
-          }}
-        >
-          <UserIcon
-            titleAccess="User"
-            sx={{ width: '16px', height: '16px' }}
-          />
-        </Box>
-        <Box
-          component="span"
-          className={identifierSpanClasses}
-          data-se="identifier"
-          title={identifier}
-          sx={{
-            marginInlineStart: '10px',
-          }}
-          aria-label={identifier}
-        >
-          {identifier}
-        </Box>
-      </Box>
+      <Tag
+        icon={(
+          <Box
+            sx={{
+              width: '16px',
+              height: '16px',
```

Comment:
Shouldn't these use design tokens?

Reply:
There is the dedicated ticket to review icon sizing: [OKTA-671459](https://oktainc.atlassian.net/browse/OKTA-671459)